backend/src/main/resources/application.yaml

```diff
@@ -78,6 +78,7 @@ management:
   endpoint.health.probes.enabled: true
   endpoint.info.enabled: true
   endpoints.web.exposure.include: health, info
+  endpoint.health.show-components: always
```

Comment:
Do we need `management.endpoint.health.show-details` as well? It looks like the [spring docs](https://docs.spring.io/spring-boot/docs/current/reference/html/actuator.html#actuator.endpoints.health) mention that if we have security enabled and want to use `always` then the security configuration must permit access to the health endpoint for both authenticated and unauthenticated users. If I'm understanding it correctly, was that what we were trying to do in the original PR [here](https://github.com/CDCgov/prime-simplereport/pull/7057/files#diff-285fb8d30ceced3e15ec5a0dc3325d035f586b239350c0a8cc0167d6ed6b0e6dL56)? Is this now resolved because we are [overriding the normal health endpoint](https://github.com/CDCgov/prime-simplereport/blob/e09d36254bfab7408cf543dfafe816d526763098/backend/src/main/java/gov/cdc/usds/simplereport/api/heathcheck/BackendAndDatabaseHealthIndicator.java#L22) which is already included in the security config?

Reply:
Thank you for breaking this down!! This was super clarifying!